packages/hadron-build/lib/run.js

```diff
@@ -61,44 +36,38 @@ function run(cmd, args, opts, fn) {
     opts = {};
   }
 
-  getBinPath(cmd, function(err, bin) {
-    if (err) {
-      return fn(err);
-    }
+  debug('running', { cmd, args });
 
-    debug('running', { cmd, args });
-
-    var output = [];
-    var proc = spawn(bin, args, opts);
-    proc.stdout.on('data', function(buf) {
-      buf.toString('utf-8').split('\n').map(function(line) {
-        debug('  %s> %s', cmd, line);
-      });
-      output.push(buf);
+  var output = [];
+  var proc = spawn(cmd, args, opts);
+  proc.stdout.on('data', function(buf) {
+    buf.toString('utf-8').split('\n').map(function(line) {
+      debug('  %s> %s', cmd, line);
     });
-    proc.stderr.on('data', function(buf) {
-      buf.toString('utf-8').split('\n').map(function(line) {
-        debug('  %s> %s', cmd, line);
-      });
-      output.push(buf);
+    output.push(buf);
+  });
+  proc.stderr.on('data', function(buf) {
+    buf.toString('utf-8').split('\n').map(function(line) {
+      debug('  %s> %s', cmd, line);
     });
+    output.push(buf);
+  });
 
-    proc.on('exit', function(code) {
-      if (code !== 0) {
-        debug('command failed!', {
-          cmd: cmd,
-          output: Buffer.concat(output).toString('utf-8')
-        });
-        fn(new Error('Command failed!  '
-          + 'Please try again with debugging enabled.'), Buffer.concat(output).toString('utf-8'));
-        return;
-      }
-      debug('completed! %j', {
-        cmd: cmd
+  proc.on('exit', function(code) {
+    if (code !== 0) {
+      debug('command failed!', {
+        cmd: cmd,
+        output: Buffer.concat(output).toString('utf-8')
       });
-
-      fn(null, Buffer.concat(output).toString('utf-8'));
+      fn(new Error('Command failed!  '
```

Comment:
Probably we want to specify here in the error message how to enable debugging?

Reply:
So, this is old code, I didn't change this bit in this PR. But I still was thinking about just changing this error to include actual output, because this has definitely been super annoying multiple times in the past.

So I've just added 1e6bfb627a799fb26dc652ee312a3eec5f9fc944 to make the error a bit more obvious here from the beginning, lmk what you think!